ansible/roles/ansible-datamover-api/tasks/install_ubuntu.yml

```diff
@@ -62,6 +62,42 @@
   notify:
     - restart tvault-datamover-api
 
+- name: enable and daemon reload tvault-datamover-api service
+  shell: systemctl daemon-reload && systemctl enable tvault-datamover-api.service
+  when: populate_conf.changed or create_systemd.changed
+  notify:
+      - restart tvault-datamover-api
+
 - name: ensure /tmp/datamover_url file is deleted
   file: path="/tmp/datamover_url" state=absent
 
+- name: Disable tvault-datamover-api service
+  service:
+    name: "{{ item }}"
+    state: stopped
```

Comment:
We should not need to stop this service. We should set correct permissions on log directory before starting the dmapi service. Please add changes.

Reply:
sure